### ISSUE TITLE
[P7/E] Build TLM fine-tuning, LiteRT conversion, and cross-device evaluation protocol

### ISSUE THREAD

**Issue by ed3c** [ed3c/ai-edge-tlm#12]
Parent: #1
Depends on: #2 source/model-license admission, #3 contracts, #9 model supply chain. Integration consumes #6/#8 runtime adapters when available.

## Goal
Create reproducible, evidence-bounded workflows for task narrowing, synthetic-data generation, SFT, model conversion, parity checks, function/tool accuracy evaluation, and device performance/thermal benchmarking.

## Owns
- `training/**`
- `eval/**`
- public recipes, schemas, synthetic non-private fixtures, benchmark harness and receipts

## Required behavior
- pin model, dataset, teacher/model-service, trainer, converter and runtime identities separately;
- private/user data is excluded from public fixtures and logs;
- synthetic data generation records prompt/template/version and train/eval split lineage;
- compare pre-training baseline, post-SFT source model, converted `.litertlm`, and device-runtime behavior;
- function/tool evaluation reports per-function as well as aggregate metrics and invalid-JSON/schema failures;
- benchmark receipts include device/OS/runtime/model/quantization/backend, TTFT, prefill/decode throughput, peak memory, energy/thermal sampling and test parameters;
- vendor/talk benchmarks remain source evidence, never copied into local-result fields.

## Acceptance gates
- deterministic toy/synthetic pipeline fixture;
- held-out/edge/noisy/multilingual and adversarial tool inputs;
- conversion parity threshold and explicit failure state;
- planted train/eval leakage and stale-model identity make the harness fail;
- device benchmark schema refuses missing backend/device/runtime identity.

## Evidence ceiling
Training loss or desktop conversion PASS cannot prove mobile task accuracy. Emulator/device performance lanes remain separate from model-quality lanes.

## Handoff
P8 consumes only model artifacts and evaluation receipts that bind the same immutable model manifest and runtime revision.

**Comment by ed3c** [comment]
## Tech Lead worker admission — P7

```text
state                   START_READY_ONLY
primary parent          models/artifact-supply-v1@6b90db1654d10cd34ad093890af4209daf810b7d
primary tree            c3c2135467e3b5578a3c25361bc776afbb8c876d
canonical head          19f79d6cc61a1b8b01ad9da77c380c3002264754
canonical tree          e094ddc51805d19349bbd4f139682917bf8d2af5
task packet blob        25d6d083045616f85d5c130316cc94216ad97d98
diff from parent        exactly 1 task-packet path
PR                      NOT_CREATED_BY_POLICY
Shadow                  CONTINUE_WITH_WARNINGS_L1
```

The connector’s initial task commit was on the cross-worker incident chain. Shadow blocked downstream execution at L3. A forward-only two-parent reconciliation now uses exact P4 as first parent and keeps the incident commit as second audit parent; the current tree is P4 plus only the P7 packet.

Packet: `training/task-packets/p7-training-conversion-v1.task.json`.

The worker may write only `training/**`, `eval/**`, and `tests/p7/**`. Toy/static work may open a Draft PR after gates; private data, real model/service access, model terms, mobile runtime parity, physical-device quality/performance/privacy/energy/thermal, merge, and release remain separate lanes.

**Comment by ed3c** [comment]
## Tech Lead exact-stage closure — P7

```text
PR #27                 Draft / Open / Mergeable / Unmerged
implementation         13e0130a26b20681cf032141dbbb333911cc2ac9 / bf08b68ead102be89b41530b80f517cc11701e93
exact head/tree        86981ba64089af8f961f2045a3d7de01cf20c469 / 5be0513363f46384072403457e22041d12180205
exact receipt blob     bf27a9ea10b34889ad7859de07a7f8eef453776b
provider CI            validate #37 / SUCCESS
Shadow                 CONTINUE_WITH_WARNINGS_L1
```

Synthetic/static/local protocol slice is implemented: deterministic lineage, leakage controls, identity-bound function metrics, invalid JSON/schema accounting, conversion parity threshold and strict device receipt schema. Vendor/talk results cannot populate local-result fields. Real SFT/conversion, private data/services, model quality, P3C/P3D runtime parity, physical devices, privacy/power/thermal, terms, merge and release remain open.

Exact receipt: `training/receipts/p7-exact.json`.
